e2e/cypress/integration/channel_sidebar/unread_filter_spec.js

```diff
@@ -214,6 +224,7 @@ function disableUnreadFilter() {
 function createChannel(teamId, channelName, message) {
     cy.apiCreateChannel(teamId, channelName, channelName, 'O', '', '', false).then(({channel}) => {
         if (message) {
+            cy.wait(TIMEOUTS.THREE_SEC);
```

Comment:
This wait time is to have an interval between channel creation and posting message to the said channel, which guarantees receiving unread message.